amplify.yml

```diff
@@ -11,9 +11,6 @@ frontend:
         - yarn install
     build:
       commands:
-        - REACT_APP_APES_AUTH_SECRET_KEY=${REACT_APP_APES_AUTH_SECRET_KEY}
-        - REACT_APP_ANGEL_AUTH_SECRET_KEY=${REACT_APP_ANGEL_AUTH_SECRET_KEY}
-        - REACT_APP_KADO_API_KEY=${REACT_APP_KADO_API_KEY}
         - REACT_APP_NETWORK=${REACT_APP_NETWORK}
```

Comment:
Can remove this from build I think since we're not using it anymore.
```suggestion
```

Reply:
checked that this env profile is still in use for these endpoints
POST `${v(1)}/fiat/stripe-proxy/apes/${network}`,
GET `/${v(5)}/endowments/${network}`
 GET `/${v(2)}/profile/${network}/endowment/${endowId}`
 GET `/${v(1)}/profile/${network}/program/${endowId}/${programId}`
DELETE |  PUT `/${v(3)}/profile/${network}/endowment`

ser @jp-mariano, let me know here the updated endpoints without network

Reply:
@jp-mariano We can also leave this one alone for now, since it's not sensitive data, it's not a high priority to spend a lot of time refactoring out and risk introducing bugs so close to the launch date. If it's just a single recent lambda to update or all those mentioned above are (or will be) depreciated post-v1.9 launch, then full steam ahead.

Reply:
POST `${v(1)}/fiat/stripe-proxy/apes/${network}`
- For now, we can keep this as is. The `stripe-proxy` Lambda would probably still be refactored and from there we can update the endpoint it uses to one without the `network` parameter.

GET `/${v(5)}/endowments/${network}`
- This is the marketplace endpoint and `network` here is still needed in order to differentiate prod from staging endowments. We can keep this for now.

GET `/${v(2)}/profile/${network}/endowment/${endowId}`
- This is the `get-single-endowment` Lambda's old endpoint which will be deprecated. The new endpoint here should be `GET /v1/profile/endowment/{id}` or `GET /staging/profile/endowment/{id}`

GET `/${v(1)}/profile/${network}/program/${endowId}/${programId}`
- This is currently a working endpoint and `network` here is needed in order to differentiate prod from staging endows. We can also keep this for now.

DELETE | PUT `/${v(3)}/profile/${network}/endowment`
- This endpoint will be deprecated and the new one should be `DELETE | PUT /v1/profile/endowment` or `DELETE | PUT`

Reply:
got this @jp-mariano